test/unit/api/tuple/swizzle/broadcast.cpp

```diff
@@ -0,0 +1,44 @@
+//==================================================================================================
+/**
+  EVE - Expressive Vector Engine
+  Copyright : EVE Contributors & Maintainers
+  SPDX-License-Identifier: MIT
+**/
+//==================================================================================================
+#include "test.hpp"
+#include <eve/wide.hpp>
+#include <bit>
+#include <cstdint>
+
+template<int I, int N>
+inline constexpr auto broadcast = eve::fix_pattern<N>( [](int, int){ return I; } );
+
+//==================================================================================================
+// Identity test
+//==================================================================================================
+EVE_TEST_TYPES( "Check behavior of broadcast swizzle", eve::test::scalar::all_types)
+<typename T>(eve::as_<T>)
```

Comment:
Why not just run these for all `simd` types and take the cardinal from there.
That way you don't need to do wto level of constexpr for_